voluptuous/encoding.py

```diff
@@ -0,0 +1,5 @@
+def to_utf8_py2(data):
```

Comment:
I'm having some trouble with this. When I move the function I'm getting lots of doctest errors.